src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooBuildPlanService.java

```diff
@@ -95,20 +97,22 @@ public BambooBuildPlanService(ResourceLoaderService resourceLoaderService, Bambo
      *                               unique identifier
      * @param solutionRepositoryName the slug of the solution repository, i.e. the
      *                               unique identifier
+     * @param auxiliaryRepositories  List of auxiliary repositories to be included in
+     *                               the build plan
      */
     public void createBuildPlanForExercise(ProgrammingExercise programmingExercise, String planKey, String repositoryName, String testRepositoryName,
-            String solutionRepositoryName) {
+                                           String solutionRepositoryName, List<Pair<String, String>> auxiliaryRepositories) {
```

Comment:
This does not look like the way we normally format the code. Does the code style accept this?

We usually only add one indentation level.